src/common/low_precision_transformations/src/fake_quantize_dequantization.cpp

```diff
@@ -123,12 +123,16 @@ bool FakeQuantizeDequantization::checkElementwise(const std::shared_ptr<ngraph::
         return false;
     }
 
-    const auto channelsDimension = partialShape[1];
+    const auto channelsDimension = partialShape[partialShape.size() > 1ul ? 1ul : 0ul];
```

Comment:
Do we cover this change by tests? Looks like we need a test-case with 1D data shape